### DISCUSSION TITLE
Table where items can span multiple rows and columns

### DISCUSSION THREAD

**Discussion by wasi-master** [Textualize/rich#1928] [Q&A]
What I want:
```
+------------+------------+-----------+
| Header 1   | Header 2   | Header 3  |
+============+============+===========+
| body row 1 | column 2   | column 3  |
+------------+------------+-----------+
| body row 2 | Cells may span columns.|
+------------+------------+-----------+
| body row 3 | Cells may  | Cells     |
+------------+ span rows. | contain   |
| body row 4 |            | blocks.   |
+------------+------------+-----------+
```
How could I achieve a table like that with rich?

**Context**: I am making [`rich-rst`](https://pypi.org/project/rich-rst) and reStructuredText tables support this format so I was hoping to show this somehow inside the terminal.

**Comment by codenotworking** [comment]
There is already an open issue for this: [https://github.com/Textualize/rich/issues/164](https://github.com/Textualize/rich/issues/164)
